noir-projects/noir-contracts/contracts/lending_contract/src/main.nr

```diff
@@ -309,13 +266,14 @@ contract Lending {
     }
 
     unconstrained fn get_asset(asset_id: Field) -> pub Asset {
-        storage.assets.at(asset_id).read()
+        let res: Asset = storage.assets.at(asset_id).read();
+        res
```

Comment:
Curious why this change was needed

Reply:
Compiler was broken at some point and I added the type binding to check if that fixed it...then promptly forgot about it. Reverted!